_getting-started/search-data.md

```diff
@@ -0,0 +1,446 @@
+
+### Full-text search
+
+You can run a full-text search on fields mapped as `text`. By default, text fields are analyzed by the `default` analyzer. The analyzer splits text into terms and makes it lowercase. For more information about OpenSearch analyzers, see [Analyzers]({{site.url}}{{site.baseurl}}/analyzers/).
+
+For example, the following query searches for students with the name `john`:
+
+```json
+GET /students/_search
+{
+  "query": {
+    "match": {
+      "name": "john"
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+The response contains the matching document:
+
+```json
+{
+  "took": 13,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 1,
+      "relation": "eq"
+    },
+    "max_score": 0.9808291,
+    "hits": [
+      {
+        "_index": "students",
+        "_id": "1",
+        "_score": 0.9808291,
+        "_source": {
+          "name": "John Doe",
+          "gpa": 3.89,
+          "grad_year": 2022
+        }
+      }
+    ]
+  }
+}
+```
+
+Notice that the query text is lowercase while the text in the field is not, but the query still returns the matching document. 
+
+You can reorder the terms in the search string. For example, the following query searches for `doe john`:
+
+```json
+GET /students/_search
+{
+  "query": {
+    "match": {
+      "name": "doe john"
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+The response contains two matching documents:
+
+```json
+{
+  "took": 14,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 2,
+      "relation": "eq"
+    },
+    "max_score": 1.4508327,
+    "hits": [
+      {
+        "_index": "students",
+        "_id": "1",
+        "_score": 1.4508327,
+        "_source": {
+          "name": "John Doe",
+          "gpa": 3.89,
+          "grad_year": 2022
+        }
+      },
+      {
+        "_index": "students",
+        "_id": "3",
+        "_score": 0.4700036,
+        "_source": {
+          "name": "Jane Doe",
+          "gpa": 3.52,
+          "grad_year": 2024
+        }
+      }
+    ]
+  }
+}
+```
+
+The match query type uses `OR` as an operator by default, so the query is functionally `doe OR john`. Both `John Doe` and `Jane Doe` matched the word `doe`, but `John Doe` is scored higher because it also matched `john`.
+
+### Keyword search
+
+The `name` field contains the `name.keyword` subfield, which was added by OpenSearch automatically. You can try to search the `name.keyword` field in a manner similar to the previous request:
+
+```json
+GET /students/_search
+{
+  "query": {
+    "match": {
+      "name.keyword": "john"
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+This request returns no hits because the `keyword` fields must be matched exactly. 
+
+However, you can search for the exact text `John Doe`:
```

Comment:
Same comment. Would this be better structured as "However, if you search for the exact text `John Doe`:

[Example]

Then OpenSearch returns..."?